Doc/Makefile

```diff
@@ -137,14 +137,21 @@ pydoc-topics: build
 htmlview: html
 	 $(PYTHON) -c "import webbrowser; webbrowser.open('build/html/index.html')"
 
-clean:
-	-rm -rf build/* $(VENVDIR)/*
+clean: clean-venv
+	-rm -rf build/*
+
+clean-venv:
+	rm -rf $(VENVDIR)
```

Comment:
Note this change:

```
- rm -rf $(VENVDIR)/*
+ rm -rf $(VENVDIR)
```

The reason for the change is that it simplifies the conditional action in the `venv` rule:

```bash
# this works but smells bad because it's arbitrarily checking only venv/bin
[ -d $(VENVDIR)/bin ]
# checking venv/bin, venv/lib, venv/include seems superfluous
[ -d $(VENVDIR) ]  # this is nice
```

However, `rm -rf`'ing an additional level higher is always worth a close review.

Reply:
I note that you're no longer ignoring the exit code here. Since `-f` is used anyway, the likelihood of a non-zero exit code from `rm` is indeed low but it can still fail due to wrong permissions. Arguably, your current version is therefore better then.